model/RdsResultStorage.php

```diff
@@ -528,12 +533,15 @@ protected function prepareVariableData($deliveryResultIdentifier, $test, Variabl
             $variable->setEpoch(microtime());
         }
 
-        return [
-            self::VARIABLES_FK_COLUMN => $deliveryResultIdentifier,
-            self::TEST_COLUMN => $test,
-            self::VARIABLE_IDENTIFIER => $variable->getIdentifier(),
-            self::VARIABLE_VALUE => $this->serializeVariableValue($variable),
-        ];
+        return array_merge(
+            [
+                self::VARIABLES_FK_COLUMN => $deliveryResultIdentifier,
+                self::TEST_COLUMN => $test,
+                self::VARIABLE_IDENTIFIER => $variable->getIdentifier(),
+                self::VARIABLE_VALUE => $this->serializeVariableValue($variable),
```

Comment:
We should benefit from this backward compatibility change to change the serialize with json encdoe

Reply:
Little impediment for changing the serialization of variable to json: variable can be OutcomeVariable, ResponseVariable, TraceVariable... So making it generic would either involve modifying the `Variable::toJson` method to store the class name, or use a serializer like https://symfony.com/doc/current/components/serializer.html. Can we skip for the moment?